### PR TITLE
Pretty printing of Maybe* types

### PR DESCRIPTION
**Pull request by p-kozlowski**

### CODE REVIEW

File: extension/goext/types.go [hunk 1]
```diff
@@ -90,6 +94,19 @@ func (m Maybe) HasValue() bool {
...
 }
 
+func (m Maybe) String() string {
+	switch m.MaybeState {
+	case MaybeNull:
+		return "<null>"
+	case MaybeUndefined:
+		return "<undefined>"
+	case MaybeValue:
+		return "<valid value>"
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/goext/types.go]: why not just valid?

File: extension/goplugin/types_test.go [hunk 2]
```diff
@@ -18,6 +18,8 @@ package goplugin_test
 import (
 	"encoding/json"
 
+	"fmt"
+
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/goplugin/types_test.go]: unnecessary enter, gofmt

File: extension/goext/types.go (hunk 1 above)
Comment (zimnx) [reply extension/goext/types.go]: Maybe(String|Int|Float|Bool) also implement String(). They will print 'just value' if they have it. Maybe type doesn't know anything about value, only state.

File: extension/goplugin/types_test.go (hunk 2 above)
Comment (p-kozlowski) [reply extension/goplugin/types_test.go]: fixed

File: extension/goext/types.go (hunk 1 above)
Comment (p-kozlowski) [reply extension/goext/types.go]: fixed

**Pull request merged**